tests/h/services/search_index/_queue_test.py

```diff
@@ -68,25 +68,12 @@ def test_add_where_with_force(
     ):
         annotation = factories.Annotation.create()
 
-        queue.add_where(
-            tag="test_tag",
-            priority=1,
-            where=[Annotation.id == annotation.id],
-            force=force,
-        )
+        queue.add_where("test_tag", 1, [Annotation.id == annotation.id], force=force)
 
-        assert db_session.query(Job).one() == Any.instance_of(Job).with_attrs(
-            {
-                "kwargs": Any.dict.containing(
-                    {
-                        "force": expected_force,
-                    }
-                )
-            }
-        )
+        assert db_session.query(Job).one().kwargs["force"] == expected_force
```

Comment:
I copied the previous way of doing it, then realised I was trying _waaay_ to hard here